conda_smithy/github.py

```diff
@@ -116,64 +124,73 @@ def create_github_repo(args):
+    current_maintainers = []
+    if not team:
+        team = create_team(
+            org,
+            team_name,
+            'The {} {} contributors!'.format(
+                choice(superlative), team_name
             )
-            team_name = meta.name()
-            # Try to get team or create it if it doesn't exist.
-            team = next((team for team in gh_repo.get_teams() if team.name == team_name), None)
-            current_maintainers = []
-            if not team:
-                team = create_team(
-                    user_or_org,
-                    team_name,
-                    'The {} {} contributors!'.format(
-                        choice(superlative), team_name
-                    )
-                )
-                team.add_to_repos(gh_repo)
-            else:
-                current_maintainers = team.get_members()
-
-
-            # Add only the new maintainers to the team.
-            current_maintainers_handles = set([
-                e.login.lower() for e in current_maintainers
-            ])
-            for new_maintainer in maintainers - current_maintainers_handles:
-                add_membership(team, new_maintainer)
-
-            # Mention any maintainers that need to be removed (unlikely here).
-            for old_maintainer in current_maintainers_handles - maintainers:
-                print(
-                    "AN OLD MEMBER ({}) NEEDS TO BE REMOVED FROM {}".format(
-                        old_maintainer, repo_name
-                    )
-                )
+        )
+        team.add_to_repos(gh_repo)
+    else:
+        current_maintainers = team.get_members()
 
-            # Get the all-members team
-            team_name = 'all-members'
-            description = "All of the awesome {} contributors!".format(user_or_org.name)
-            all_members_team = get_cached_team(user_or_org, team_name, description)
-
-            # Add new members to all-members
-            for new_member in maintainers - current_maintainers_handles:
-                if not has_in_members(all_members_team, new_member):
-                    print(
-                        "Adding a new member ({}) to {}. Welcome! :)".format(
-                            new_member, user_or_org.name
-                        )
-                    )
-                    add_membership(all_members_team, new_member)
 
+    # Add only the new maintainers to the team.
+    current_maintainers_handles = set([
+        e.login.lower() for e in current_maintainers
+    ])
+    for new_maintainer in maintainers - current_maintainers_handles:
+        add_membership(team, new_maintainer)
+
+    # Mention any maintainers that need to be removed (unlikely here).
+    for old_maintainer in current_maintainers_handles - maintainers:
+        print(
+            "AN OLD MEMBER ({}) NEEDS TO BE REMOVED FROM {}".format(
+                old_maintainer, repo_name
+            )
+        )
```

Comment:
Are we planning to handle removals here. Should add that we should be standardize casing all usernames before this check as GitHub is case insensitive and tends to switch to lower. So some might not match even though they are in both groups.

Reply:
Both are lower-cased. See https://github.com/conda-forge/conda-smithy/pull/565/files#diff-4970770929969f2ad4e818ed6338e5b1R146 and https://github.com/conda-forge/conda-smithy/pull/565/files#diff-4970770929969f2ad4e818ed6338e5b1R166